src/core/example_settings.py

```diff
@@ -1,464 +1,24 @@
-            'format': '%(levelname)s %(asctime)s %(module)s '
-            'P:%(process)d T:%(thread)d %(message)s',
-        },
-        'coloured': {
-            '()': 'colorlog.ColoredFormatter',
-            'format': '%(log_color)s%(levelname)s %(asctime)s %(module)s '
-            'P:%(process)d T:%(thread)d %(message)s',
-            'log_colors' : {
-                'DEBUG':    'green',
-                'WARNING':  'yellow',
-                'ERROR':    'red',
-                'CRITICAL': 'red',
-            }
-        },
-    },
-    'handlers': {
-        'console': {
-            'level': 'DEBUG',
-            'class': 'logging.StreamHandler',
-            'formatter': 'coloured',
-            'stream': 'ext://sys.stdout',
-        }
-    },
-    'loggers': {
-        'django.db.backends': {
-            'level': 'WARNING',
-            'handlers': ['console'],
-            'propagate': False,
-        },
-    },
-}
-
-
-class SuppressDeprecated(logging.Filter):
-    def filter(self, record):
-        WARNINGS_TO_SUPPRESS = [
-            'RemovedInDjango110Warning',
-        ]
-        # Return false to suppress message.
-        return not any([warn in record.getMessage() for warn in WARNINGS_TO_SUPPRESS])
-
-
-MESSAGE_TAGS = {
-    messages.ERROR: 'danger',
-}
-
-LOGIN_REDIRECT_URL = '/user/profile/'
-LOGIN_URL = '/login/'
-
-EMAIL_BACKEND = 'django.core.mail.backends.smtp.EmailBackend'
-#EMAIL_BACKEND = "django.core.mail.backends.console.EmailBackend"
-EMAIL_HOST = ''
-EMAIL_PORT = ''
-EMAIL_HOST_USER = ''
-EMAIL_HOST_PASSWORD = ''
-EMAIL_USE_TLS = True
-
-# Settings for use with Mailgun
-MAILGUN_ACCESS_KEY = ''
-MAILGUN_SERVER_NAME = ''
-MAILGUN_REQUIRE_TLS = False
-ENABLE_ENHANCED_MAILGUN_FEATURES = False  # Enables email tracking
-
-
-DATE_FORMT = "Y-m-d"
-DATETIME_FORMAT = "Y-m-d H:i"
-
-AUTH_USER_MODEL = 'core.Account'
-
-PLUGIN_HOOKS = {}
-
-NOTIFY_FUNCS = []
-
-ENABLE_ORCID = True
-ORCID_API_URL = 'http://pub.orcid.org/v1.2_rc7/'
-ORCID_URL = 'https://orcid.org/oauth/authorize'
-ORCID_TOKEN_URL = 'https://pub.orcid.org/oauth/token'
-ORCID_CLIENT_SECRET = ''
-ORCID_CLIENT_ID = ''
-
-
-SESSION_COOKIE_NAME = 'JANEWAYSESSID'
-
-S3_ACCESS_KEY = ''
-S3_SECRET_KEY = ''
-S3_BUCKET_NAME = ''
-END_POINT = 'eu-west-2'  # eg. eu-west-1
-S3_HOST = 's3.eu-west-2.amazonaws.com'  # eg. s3.eu-west-1.amazonaws.com
-
-BACKUP_TYPE = 'directory'  # s3 or directory
-BACKUP_DIR = '/path/to/backup/dir/'
-BACKUP_EMAIL = False  # If set to True, will send an email each time backup is run
-
-URL_CONFIG = 'domain'  # path or domain
-
-# Captcha
-# You can get reCaptcha keys for your domain here: https://developers.google.com/recaptcha/intro
-# You can set either to use Google's reCaptcha or a basic math field with no external requirements
-
-INSTALLED_APPS.append('snowpenguin.django.recaptcha2')
-
-CAPTCHA_TYPE = 'recaptcha'  # should be either simple_math or recaptcha to enable captcha fields otherwise disabled
-RECAPTCHA_PRIVATE_KEY = ''
```

Comment:
Probably a good idea to have the captcha settings over in example as they need configuring.